scala/src/main/scala/com/concord/wordcounter/WordCounter.scala

```diff
@@ -0,0 +1,54 @@
+package com.concord.wordcounter
+
+import java.util
+
+import com.concord._
+import com.concord.swift._
+
+class WordCounter extends Computation {
+
+  private val histogram: util.HashMap[String, Integer] = new util.HashMap()
```

Comment:
Will take a little more time to "scalaify" this code since a lot of its features were ported from the Java example.